services/web/server/src/simcore_service_webserver/login/_auth_handlers.py

```diff
@@ -123,25 +158,25 @@ async def login(request: web.Request):
                     "message": MSG_PHONE_MISSING,
                     "next_url": f"{request.app.router['auth_register_phone'].url_for()}",
                 },
-                # TODO: deprecated: remove in next PR with @odeimaiz
+                # NOTE: REMOVE when frontend is refactored
```

Comment:
please create an issue to follow up on this and add it in the note thanks

Reply:
Odei is now working on refactoring the frontend with this branch, so I will remove it straight after his commits go in.